File: mcspring-build/mcspring-plugin-manager/src/test/java/in/kyle/mcspring/manager/commands/TestCommandHeal.java
package in.kyle.mcspring.manager.commands;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;

import java.util.List;

import in.kyle.mcspring.test.MCSpringTest;
import in.kyle.mcspring.test.command.TestCommandExecutor;

import static org.assertj.core.api.Assertions.assertThat;

@MCSpringTest
class TestCommandHeal {

    @Autowired
    TestCommandExecutor commandExecutor;
    
    @Test
    void testHeal() {
        List<String> output = commandExecutor.run("heal");
        assertThat(output).first().asString().matches("Healed [^ ]+");
    }
}

File: mcspring-build/mcspring-plugin-manager/src/test/java/in/kyle/mcspring/manager/commands/TestCommandHeal.kt
package `in`.kyle.mcspring.manager.commands

import `in`.kyle.mcspring.test.MCSpringTest
import `in`.kyle.mcspring.test.command.TestCommandExecutor
import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired

@MCSpringTest
internal class TestCommandHeal {

    @Autowired
    lateinit var commandExecutor: TestCommandExecutor

    @Test
    fun testHeal() {
        val output = commandExecutor.run("heal")
        Assertions.assertThat(output).first().asString().matches("Healed [^ ]+")
    }
}
